man/git-feature.md

```diff
@@ -2,42 +2,59 @@
 
 ## SYNOPSIS
 
-`git-feature` [-a|--alias branch_prefix] [-s|--separator branch_separator] [-r|--remote [remote_name]] &lt;name&gt;  
-`git-feature` [-a|--alias branch_prefix] [-s|--separator branch_separator] finish [--squash] &lt;name&gt;
+`git-feature` [-a|--alias <PREFIX>] [-s|--separator <SEPARATOR>] [-r|--remote [REMOTE_NAME]] [--from START_POINT] <NAME>...
+
+`git-feature` [-a|--alias <PREFIX>] [-s|--separator <SEPARATOR>] finish [--squash] <NAME>...
 
 ## DESCRIPTION
 
-Create/Merge the given feature branch
+Create or merge the given feature branch. The feature branch name is made from the <PREFIX>, the <SEPARATOR>, and the <NAME> joined together.
+
+The default <PREFIX> is `feature` and <SEPARATOR> is `/`, which can be changed (see OPTIONS and GIT CONFIG for details).
+
+The branch <NAME> may be specified as multiple words which will be joined with `-`. If the branch name contains the word `finish` or is another OPTION, `--` should be passed to stop OPTION parsing. See the EXAMPLES for details.
 
 ## OPTIONS
 
-&lt;-a|--alias branch_prefix&gt;
+- `-a` <PREFIX>, `--alias` <PREFIX>:
+
+  The branch prefix to use, or `feature` if not supplied.
+
+- `-s` <SEPARATOR>, `--separator` <SEPARATOR>:
+
+  The separator to use for joining the branch prefix and the branch name, or `/` if not supplied.
+
+- `-r` [REMOTE_NAME], `--remote` [REMOTE_NAME]:
```

Comment:
I wonder why `[]` is used here while `<>` is used to wrap the argument above.

Reply:
Because `REMOTE_NAME` is optional; it does not require the second parameter but defaults to `origin` if just `-r` is provided. (As I’m reading it, `git create-branch` takes `-r` to create the branch and set its upstream as `REMOTE_BRANCH`. So, `-r` is just a pass-through to `git-create-branch(1)`.

Reply:
Thanks for your explanation